### ISSUE TITLE
Eclipses of Jupiter's Moons

### ISSUE THREAD

**Issue by gzotti** [Stellarium/stellarium#222]
A very nitpickish issue, admittedly:

### Expected Behaviour
Mutual eclipses of Jupiter's Moons should be on time

### Actual Behaviour
Those eclipses are about half hour wrong

### Steps to reproduce
- Observe Christopher Go's eclipse record on https://www.universetoday.com/115814/observing-challenge-catch-a-series-of-mutual-eclipses-by-jupiters-moons/ -- actual file URL https://www.universetoday.com/wp-content/uploads/2014/11/gany-io20090816.gif
- Compare with simulated view, 2009-08-16 17:09UT
- Observe a few tens of minutes difference (although I cannot stop the GIF to see exactly, but around 25 minutes or so?)

### System
* Stellarium version: 0.18.1 or build master.16115
* Operating system: Windows 10 (should not matter)
* Graphics Card: Geforce 960M, irrelevant

### Logfile 
Not relevant

I assume that maybe shadow computation has some error around light time correction, although I must admit some of the details in the Planet class are very hard to follow. The geometrical accuracy seems OK, compare time of minimum separation (with light time correction active) in the GIF video vs. simulation.

**Comment by alex-w** [comment]
Fun. It looks like light speed correction is not applied to the shadow.

**Comment by guillaumechereau** [comment]
Could the problem actually be in the position of the moons?  From the same reference page, the  December 21, 2014 occultation of Io by Ganymede also doesn't seem to match exactly (about 10 min difference).

**Comment by guillaumechereau** [comment]
I did a branch to update the moons model from L1 to L1.2:
https://github.com/Stellarium/stellarium/tree/jupiter-moons-l12-model

It seems to fix this particular event.

I think we should test it out a bit further, and if there is no issue we could merge it and remove the L1 algo code.

**Comment by gzotti** [comment]
Oh, great! I was not even aware of L1.2. Indeed, the time fits very well.  If this is the currently recommended model and else a drop-in replacement for L1.1, sure, please go ahead and merge.
